styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/DockerRunnerHandlerTest.java

```diff
@@ -164,36 +178,6 @@ public void shouldHaltIfMissingExecutionDescription() throws Exception {
     assertThat(stateManager.get(workflowInstance), is(nullValue()));
   }
 
-  @Test
-  public void shouldPerformCleanupOnFailed() throws Exception {
```

Comment:
Should these tests instead be updated to verify that cleanup happens after grace period?

Reply:
As we talked, we can't do that here due to the DockerRunner being mocked. But the isolated  behaviour is tested in KubernetesDockerRunnerTest